opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -102,6 +103,34 @@ def get_context(self) -> 'SpanContext':
             A :class:`.SpanContext` with a copy of this span's immutable state.
         """
 
+    def set_attribute(self: 'Span',
+                      key: str,
+                      value: 'types.AttributeValue'
+                      ) -> None:
+        """Sets an Attribute.
+
+        Sets a single Attribute with the key and value passed as arguments.
+        """
+
+    def add_event(self: 'Span',
+                  name: str,
+                  attributes: 'types.Attributes',
+                  ) -> None:
+        """Adds an Event.
+
+        Adds a single Event with the name and, optionally, attributes passed
+        as arguments.
+        """
+
+    def add_link(self: 'Span',
+                 context: 'SpanContext',
+                 attributes: 'types.Attributes',
+                 ) -> None:
+        """Adds a Link to another Span.
+
+        Adds a single Link to the Span via the SpanContext passed as argument.
```

Comment:
```suggestion
        Adds a single `Link` to the `Span` via the `SpanContext` passed as argument.
```

Reply:
"the Span" should be "the span" (following your rationale https://github.com/open-telemetry/opentelemetry-python/pull/83#discussion_r313568783)